src/osm_configurator/view/popups/yes_no_pop_up.py

```diff
@@ -27,37 +37,73 @@ def __init__(self, message: str, func):
             message (str): The message to be shown in the PopUp.
             func (typing.Callable): Function taking one Boolean and has no return for the PopUp to return a message
         """
-        self.func = func
-
-        popup = super().__init__()
-        self.geometry(POPUPSIZE)
-
-        self.title("Alert")
-
-        customtkinter.CTkLabel(popup, text=message) \
-            .pack(side="top", fill="both", expand="True", padx=10, pady=10)
-
-        customtkinter.CTkButton(self, text="Accept", command=combine_funcs(self.accept, self.destroy)) \
-            .pack(side="bottom", padx=10, pady=10)
-
-        self.button = customtkinter.CTkButton(self, text="Cancel",
-                                              command=combine_funcs(self.cancel, self.destroy)) \
-            .pack(side="bottom", padx=10, pady=10)
 
-    def accept(self):
+        self._func = func
+
+        super().__init__(
+            fg_color=frame_constants_i.FrameConstants.HEAD_FRAME_FG_COLOR.value,
+        )
+
+        # Configuring the grid
+        self.grid_columnconfigure(0, weight=1)  # Topbar: showing nothing
+        self.grid_columnconfigure(1, weight=4)  # Shows the error message
+        self.grid_columnconfigure(2, weight=2)  # Displays the buttons
+        self.grid_rowconfigure(0, weight=1)
+        self.grid_rowconfigure(1, weight=5)
+        self.grid_rowconfigure(2, weight=1)
+
+        customtkinter.CTkLabel(master=self,
```

Comment:
not a 100% sure if you an create a button and grid it at the same time, but if it works no problemo